### ISSUE TITLE
[Bug]: Missing 95% HDI's from ANOVA's

### ISSUE THREAD

**Issue by Wouterwolf** [jasp-stats/jasp-issues#2119]
### JASP Version

0.17.1

### JASP Module

ANOVA

### What analysis are you seeing the problem on?

AN(C)OVA

### What OS are you seeing the problem on?

Windows 11

### Bug Description

In the newer versions of JASP, when I conduct a Bayesian ANOVA and check the descriptives, the 95% credible intervals of the means for the groups no longer show up (the table stops after the coefficient of variation). 

![Screenshot 2023-05-03 183003](https://user-images.githubusercontent.com/35498584/235980547-2d342359-251f-463b-a5df-c8ec3733dcad.png)
Similarly (although I am not sure this next one used to be the case), when I do a Bayesian ANCOVA with 1 fixed factor and 1 covariate, I can make a descriptive plot of the regression lines per factor, but I cannot actually find a way for JASP to give me the regression coefficient (or their HDI's) of these regression lines, even though they are apparently being computed for the creation of the graph.

### Expected Behaviour

At least for the ANOVA descriptives, I expected two more columns with an 95% credible interval min and max for the means for each group.

### Steps to Reproduce

1. Open JASP with a dataset
2. Start an ANOVA (or ANCOVA)
3. Insert a factor (and a coveriate for the ANCOVA)
4. Check the descriptives box (and make sure the credible interval box is set to 95%)
5. run the analysis
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
@Wouterwolf 
At least the CIs for the BANOVAs descriptives table are restored with jasp 0.18.3
But the missing regression coefficient (or their HDI's) in BANCOVA are still a thing, it seems.

Can you confirm?

**Comment by Wouterwolf** [comment]
Yes, correct. The group means were a bug because they were there before, but the regression coefficients have never been, but would be a really nice feature, especially given that the lines themselves (+ 95% HDI's) are already visually obtainable in a graph)

**Comment by tomtomme** [comment]
From the duplicate: 
![image](https://github.com/jasp-stats/jasp-issues/assets/4428185/7adc5767-e790-420b-bc7a-290a4495edcb)
The graph that shows the ANCOVAs slopes and HDIs, which values Wouterwolf would like to have in table form. This is probably a request for classic and Bayesian ANCOVA.
